src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,25 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  max-width: ($cell-size * 7) + 10px;
```

Comment:
What is 7 and 10 px?
Would be great to create variables for all "magic numbers"